src/gateway/handlers/inviteDelete.ts

```diff
@@ -15,7 +15,10 @@ export const inviteDelete: GatewayEventHandler = async (
   const cachedInvite = await guild.invites.get(d.code)
 
   // Should not happen but here we go
-  if (cachedInvite === undefined) return
-
-  gateway.client.emit('inviteDelete', cachedInvite)
+  if (cachedInvite === undefined) {
+    return gateway.client.emit('inviteDeleteUncached', cachedInvite)
```

Comment:
Won't it be better to emit with some more details? Such as a PartialInvite interface containing channel, guild, and invite code perhaps?